src/grammar/test/case-25.tact

```diff
@@ -1,3 +1,18 @@
 fun function(a: Int, b: Int): Int {
     return (a >> b) || (a << (32 - b));
+}
+
+fun anotherFunction(
+    a: Int,
+    b: Int,
+    ): Int {
+    return (a >> b) || (a << (32 - b));
+}
+
+fun oneMoreFunction(
```

Comment:
Ah, sorry, I wasn't perhaps clear enough with the above comment. We don't need more function definitions or declarations, just function calls, something like this:

```tact
fun oneMoreFunction(): Int {
    return foo(                   // <-- function call
        x,
        y,
    );
}
```